templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+<ul>
+  <li>
+    <a href="{% url 'taxi:index' %}">Home page</a>
+  </li>
+  <li>
+    <a href="#">Manufacturers</a>
```

Comment:
The link for 'Manufacturers' is currently a placeholder ('#'). If you have a specific URL or view for the Manufacturers page, consider updating the href attribute to point to the correct URL using the Django template tag {% url %}.